lib/errors.ts

```diff
@@ -140,10 +140,14 @@ const messages: {
 	BalenaExpiredToken: () => stripIndent`
 		Looks like the session token has expired.
 		Try logging in again with the "balena login" command.`,
+
+	BalenaAmbiguousDevice: () => stripIndent`
+		Error: Multiple devices found whose UUID starts with the provided UUID.`,
 };
 
 const EXPECTED_ERROR_REGEXES = [
 	/^BalenaAmbiguousApplication/, // balena-sdk
+	/^BalenaAmbiguousDevice/, // balena-sdk
```

Comment:
Bumping balena-errors and running `npm dedupe` might allow you to drop this.

Reply:
`EXPECTED_ERROR_REGEXES` contains regular expressions for a selected list of errors that should not be reported to Sentry.io (in addition to errors that are subclasses of the CLI's own `ExpectedError` class). I don't see how bumping `balena-errors` would remove the need of listing `BalenaAmbiguousDevice` in this array.